src/components/LoginPage/index.js

```diff
@@ -170,17 +222,38 @@ class LoginPage extends React.Component {
 
               <PrimaryButton
                 label={loading ? <Spinner /> : "login"}
+                className="LoginButton" 
                 onClick={this.handleSubmit}
+              /> 
+            </div>
+          </form>
+          <div className="LowerLoginSection">
+          <div>
+                 <p className="LoginWith"><span>Or Login with</span></p>  
+               </div> 
+               <a
+                href={gitLoading ? 0 :`${GIT_REDIRECT_URL}`}
```

Comment:
Please remove this link and add an `onClick` listener to the button. One that will go to git auth

Reply:
done